src/passthrough/mod.rs

```diff
@@ -333,14 +333,16 @@ struct HandleData {
     inode: Inode,
     file: File,
     lock: Mutex<()>,
+    open_flags: u32,
```

Comment:
How about using `AtomicU32` here so we can avoid `RwLock` in `Arc<RwLock<HandleData>>>`?

Reply:
Understood, I have pushed this suggestion